### PR TITLE
Adding software support for TI INA237 & Linear LTC3815 and update to _Sa56004x.py

### PR DESCRIPTION
**Pull request by ruck314**
### Description
- [adding python/surf/devices/ti/_Ina237.py](https://github.com/slaclab/surf/commit/50cb253d3227cb332eac51e0b5fcf2223c9a4f4c)
- [adding python/surf/devices/linear/_Ltc3815.py](https://github.com/slaclab/surf/commit/a20e528b85887a7822e8ec74980768d041fbbdfe)
- [adding dynamicAddr arg (default=False) to _PMBus.py](https://github.com/slaclab/surf/commit/240ed88f14f2f86a14a6de4be0c65515b16f1a69)
- [adding simpleViewList arg to _Sa56004x.py (similiar to _Xadc.py)](https://github.com/slaclab/surf/pull/1158/commits/e4d6e6609277cfb00c133e369aa4af1d4cd82aac)

### CODE REVIEW

File: python/surf/devices/linear/_Ltc3815.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+        self.add(pr.LinkVariable(
+            name         = 'Vin',
+            mode         = 'RO',
+            units        = 'V',
+            typeStr      = "Float32",
+            disp         = '{:1.3f}',
+            linkedGet    = lambda: self.READ_VIN.value()*4.0E-3, # Conversion factor: 4mV/Bit
```
Comment (bengineerd) [inline python/surf/devices/linear/_Ltc3815.py:25]: This should be `lambda read: self.READ_VIN.get(read=read)`. Same for all the others.

This allows rogue to do a hardware fetch when you call `self.Vin.get(read=True)` in a script.

**Review by bengineerd — COMMENTED** [review]
See code comment.

File: python/surf/devices/linear/_Ltc3815.py (hunk 1 above)
Comment (ruck314) [reply python/surf/devices/linear/_Ltc3815.py]: Fixed in [345b245](https://github.com/slaclab/surf/pull/1158/commits/345b245b83fbd3dd6ea0244375ccec4a3d28531e)

File: python/surf/devices/ti/_Ina237.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+            disp         = '{:1.3f}',
+            units        = 'A',
+            dependencies = [self.ADCRANGE,self.VSHUNT,self.senseRes],
+        ))
+
+        self.add(pr.LinkVariable(
+            name         = 'Pin',
+            description  = 'Power Measurement',
+            mode         = 'RO',
+            linkedGet    = lambda: (self.Vin.value())*(self.Iin.value()),
```
Comment (bengineerd) [inline python/surf/devices/ti/_Ina237.py:223]: Still a few .value() calls in linkedGet in this file.

Comment (ruck314) [reply python/surf/devices/ti/_Ina237.py]: Fixed [c5eddd7](https://github.com/slaclab/surf/pull/1158/commits/c5eddd76eb186f1b7602d9b1d7eecd5234f269c7)

**Review by bengineerd — APPROVED** [review]

**Pull request merged**